static/app/views/issueDetails/groupRelatedIssues/index.tsx

```diff
@@ -70,18 +75,48 @@ function GroupRelatedIssues({params}: Props) {
             message={t('Unable to load related issues, please try again later')}
             onRetry={refetch}
           />
-        ) : groups ? (
-          <GroupList
-            endpointPath={`/organizations/${orgSlug}/issues/`}
-            orgSlug={orgSlug}
-            queryParams={{query: `issue.id:[${groups}]`}}
-            query=""
-            source="related-issues-tab"
-            renderEmptyMessage={() => <Title>No related issues</Title>}
-            renderErrorMessage={() => <Title>Error loading related issues</Title>}
-          />
         ) : (
-          <b>No related issues found!</b>
+          <div>
+            <div>
+              <HeaderWrapper>
+                <Title>{t('Issues caused by the same root cause')}</Title>
+                {sameRootCauseIssues ? (
+                  <GroupList
+                    endpointPath={`/organizations/${orgSlug}/issues/`}
+                    orgSlug={orgSlug}
+                    queryParams={{query: `issue.id:[${sameRootCauseIssues}]`}}
+                    query=""
+                    source="related-issues-tab"
+                  />
+                ) : (
+                  <small>No issues caused by the same root cause.</small>
```

Comment:
We'll want to wrap this and the `small` below in the `t()` function to allow it to translate for other languages.